### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by ChristianHirsch** [tuw-cpsg/ble_client_ros#1]
Write documentation, including the dependencies, etc.

**Comment by TOTON95** [comment]
Greetings,

I found this implementation in ROS of a BLE client, and so far I installed it using libgatt as a dependency, and I fell in love with your re-connect feature, I am not able to send (I send the messages to the topic but I am not able to verify if they are right) or read (no clue at all) values, also I don't know how to register the notifications, only how to activate them but no how to specify it.  

Please, can you provide any example to complete and/or a brief quick guide would be even great? 

P.s. I am willing to help in the development of this project and open the BLE world to ROS once for all.

Thanks in advance.

**Comment by ChristianHirsch** [comment]
Hi @TOTON95 
we are working on the improvement of the documentation, especially on guides on how to install and use it. But it might take some more days. I'll inform you as soon as we have more.
Best

**Comment by TOTON95** [comment]
Hello @ChristianHirsch,

Thanks for your response, I will wait a little more then, meanwhile I am going to continue to  analyze the code a little bit deeper.

Thanks in advance.

Greetings.

**Comment by ChristianHirsch** [comment]
Hi @TOTON95, can you check out exampleSpektrumRC.md and tell me if this is sufficient for you?

**Comment by TOTON95** [comment]
@ChristianHirsch, thank you!!!, I will give it a try in these days, also thanks for add the arduino example. I saw that you made some changes to the node code, using  a direct function instead. Hopefully, meanwhile I am doing my tests, I will encounter with the bug and find a way to remove it (I am interested in getting the C/C++ implementation), i will post it if founded. 

I will keep in touch with you.

Greetings.

**Comment by jagomo** [comment]
Hi @ChristianHirsch, could it be that the "Setting up gattlib" part has an error? There is no libgatt folder in the repo described.

Kind regards.

**Comment by TOTON95** [comment]
@jagomo, indeed I forgot to mention that since I figured out my way to install libgatt from other source

**Comment by ChristianHirsch** [comment]
@jagomo yes, you are right, there were some repo mismatches. I updated the file. Can you confirm that it works now?

**Comment by jagomo** [comment]
@ChristianHirsch Yup, works like a charm now. At least up to the catkin_make part :)
